geodepy/convert.py

```diff
@@ -34,20 +34,26 @@ def __repr__(self):
     def __add__(self, other):
         return dec2dms(self.dec() + other.dec())
 
+    def __radd__(self, other):
```

Comment:
Not sure what this does?

Reply:
This method is called Reverse Add and fixed an issue with summing together multiple DMSAngle Objects. There's a great explanation here:
https://www.reddit.com/r/learnpython/comments/3cvgpi/can_someone_explain_radd_to_me_in_simple_terms_i/

Reply:
If you are adding two DMSAngle objects then both should have the add method and this shouldn't be an issue?